src/Servers/HttpSys/test/FunctionalTests/Http2Tests.cs

```diff
@@ -586,7 +587,7 @@ public async Task Reset_Http1_NotSupported()
 
         [ConditionalFact]
         [MinimumOSVersion(OperatingSystems.Windows, WindowsVersions.Win10, SkipReason = "Http2 requires Win10")]
-        [MaximumOSVersion(OperatingSystems.Windows, WindowsVersions.Win10_20H1, SkipReason = "This is last version without Reset support")]
+        [MaximumOSVersion(OperatingSystems.Windows, WindowsVersions.Win10_20H2, SkipReason = "This is last version without Reset support")]
```

Comment:
nit: The `SkipReason` makes sense only after seeing `WindowsVersions.Win10_20H2` (`10.0.19042`) is earlier than `10.0.19529`. Could you make "this" more clear without hunting around❔

Reply:
`10.0.19529` is only a preview placeholder. We don't know what the real release version will be. We'll update it after the next windows release.